src/Order.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = ({ orderType, img }) => {
+  let [orders, setOrders] = useState(0);
+  const orderOne = () => {
+    setOrders(orders + 1);
+  };
+  const minusOne = () => {
+    setOrders(orders - 1);
+  };
+  return (
+    <li className="order">
+      <img src={img} alt="food-icon" id="food-icon" />
+      <p>
+        {orderType}: {orders}
+      </p>
+      <div class="btn-group" role="group" aria-label="food-icons">
+        <RestaurantButton
+          orderOne={orderOne}
+          minusOne={minusOne}
+          className="restaurant-btn"
+        />
+      </div>
+    </li>
+  );
+};
+
+export default Order;
```

Comment:
Nice use of imgs in this component, looks good! You could set the `alt=""` here as you already have a text description for the order type & the img doesn't offer any additional information (screen readers will then skip the img). Your add/remove buttons work really nicely - you could add some logic into your minusOne func to prevent orders going below 0